boards/arm/nrf5340dk_nrf5340/CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ if (CONFIG_BUILD_WITH_TFM)
 			COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_PROPERTY:tfm,BL2_BIN_FILE> ${CMAKE_BINARY_DIR}/mcuboot.bin
 
 			# Generate an intel hex file from the signed output binary
-			COMMAND srec_cat ${CMAKE_BINARY_DIR}/tfm_sign.bin
+			COMMAND ${SREC_CAT} ${CMAKE_BINARY_DIR}/tfm_sign.bin
```

Comment:
nothing wrong here, but 55d587fed3583c758f7f0fe2d31e48a6205e11cc should now be a `[nrf fromtree]` as the upstream PR has been merged.

Reply:
Done

Reply:
Fixed